src/interface.ts

```diff
@@ -47,8 +49,13 @@ export namespace UI {
             if (args) {
                 const decompileJavaIndex = args.indexOf("decompile_java");
                 const quarkAnalysisIndex = args.indexOf("quark_analysis");
+                const jadxOptionsIndex = args.indexOf("--deobf");
                 let decompileJava = false;
                 let quarkAnalysis = false;
+                let jadxArgs: string[] = [];
+                if (jadxOptionsIndex > -1) {
+                    jadxArgs = args.splice(jadxOptionsIndex);
```

Comment:
Only remove one item here. We might forget it when we add something after it.
```suggestion
                    jadxArgs = args.splice(jadxOptionsIndex, 1);
```

Reply:
If you want.
I was in the idea to keep args separated, first args for `apktool` then args for `jadx`.
Then without specifing this number we take by default all options after `--deobf` option for `jadx`.
If we add more options for `jadx` we do not have to forget to change this value.
Just my thought.